### DISCUSSION TITLE
Welcome to messenger-clone Discussions!

### DISCUSSION THREAD

**Discussion by meMANNY** [meMANNY/Piper#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meMANNY** [comment]
How to get started with react

**Reply by AvijeetJain** [reply]
React is a JavaScript library for building user interfaces.

Open a terminal(Windows Command Prompt or PowerShell).
Create a new project folder: mkdir ReactProjects and enter that directory: cd ReactProjects .
Install React using create-react-app, a tool that installs all of the dependencies to build and run a full React.js application:

**Reply by meMANNY** [reply]
k thanks
